vizro-core/docs/pages/user_guides/custom_components.md

```diff
@@ -214,12 +127,13 @@ vm.Parameter.add_type("selector", TooltipNonCrossRangeSlider)
         ```
 
         1.  Here we provide a new type for the new component, so it can be distinguished in the discriminated union.
-        2.  Here we chose to mostly copy the source code of the build method and alter it directly. Alternatively we could overload the `build` method and alter the output of `super().build()`, but then we recommend pinning the version of `Vizro` in case of future breaking changes.
-        3.  This is the change that makes the `RangeSlider` not cross itself when moving the handle.
-        4.  This is the change that displays the tooltip below the handle.
-        5.  **Don't forget!** If part of a discriminated union, you must add the new component to the parent model where it will be inserted. In this case the new `TooltipNonCrossRangeSlider` will be inserted into the `selector` argument of the `Filter` model, and thus must be added as an allowed type.
-        6.  **Don't forget!** If part of a discriminated union, you must add the new component to the parent model where it will be inserted. In this case the new `TooltipNonCrossRangeSlider` will be inserted into the `selector` argument of the `Parameter` model, and thus must be added as an allowed type.
-        7.  The new component can now be inserted into a regular dashboard.
+        2.  Here we overload the `build` method by altering the output of `super().build()`. Alternatively one could copy the source code of the build method and alter it directly.
+        3.  `range_slider_build_obj[self.id]` then fetches the underlying `dcc.range_slider` object.
+        4.  This is the change that makes the `RangeSlider` not cross itself when moving the handle.
+        5.  This is the change that displays the tooltip below the handle.
+        6.  **Don't forget!** If part of a discriminated union, you must add the new component to the parent model where it will be inserted. In this case the new `TooltipNonCrossRangeSlider` will be inserted into the `selector` argument of the `Filter` model, and thus must be added as an allowed type.
+        7.  **Don't forget!** If part of a discriminated union, you must add the new component to the parent model where it will be inserted. In this case the new `TooltipNonCrossRangeSlider` will be inserted into the `selector` argument of the `Parameter` model, and thus must be added as an allowed type.
+        8.  The new component can now be inserted into a regular dashboard.
```

Comment:
Mostly minor rephrasing suggestions:

```suggestion
        2.  Here, we overload the `build` method by altering the output of `super().build()`. Alternatively, one could copy the source code of the build method and change it directly.
        3.  `range_slider_build_obj[self.id]` then fetches the underlying `dcc.range_slider` object.
        4.  This change prevents the `RangeSlider` from crossing itself when moving the handle.
        5.  This is the change that displays the tooltip below the handle.
        6 **Remember!** If part of a discriminated union, you must add the new component to the parent model where it will be inserted. In this case, the new `TooltipNonCrossRangeSlider` will be inserted into the `selector` argument of the `Filter` model and thus must be added as an allowed type.
        7 **Remember!** If part of a discriminated union, you must add the new component to the parent model where it will be inserted. In this case, the new `TooltipNonCrossRangeSlider` will be inserted into the `selector` argument of the `Parameter` model and thus must be added as an allowed type.
        8.  The new component can now be inserted into a regular dashboard.
```